src/resources/assets/js/store/locale.js

```diff
@@ -15,7 +15,19 @@ export const getters = {
         }
 
         const { lang } = rootState.user.preferences.global;
-        return state.i18n[lang] ? (state.i18n[lang][key] || key) : key;
+        const { env } = rootState.meta;
+        if (state.i18n[lang] && state.i18n[lang][key] !== undefined) {
+            return (state.i18n[lang][key] || key);
+        } else {
```

Comment:
the else is redundant because if the condition is true we return a value and we don't go past line 20. If the condition is false then we go to 21 with or without else.